pkg/gatewayserver/gatewayserver.go

```diff
@@ -206,6 +214,24 @@ func New(c *component.Component, conf *Config, opts ...Option) (gs *GatewayServe
 
 	hooks.RegisterUnaryHook("/ttn.lorawan.v3.NsGs", cluster.HookName, c.ClusterAuthUnaryHook())
 
+	for name, prefix := range gs.forward {
+		str := strings.Split(name, ":")
```

Comment:
Why? `SplitN` doesn't return an error so I can't enforce `type:name`.

Reply:
As far as I remember, the original config was an address, which could be IPv6 or host:port. By splitting everything (instead of just splitting the `type` of `type:rest`) you'll lose that information.

Reply:
Ah I see. Didn't know that. Updated.